template/composer.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "zendframework/<project-name>",
+    "description": "<project-description>",
+    "homepage": "https://docs.zendframework.com/<project-name>/",
+    "type": "library",
+    "license": "BSD-3-Clause",
+    "keywords": [
+        "http",
+        "middleware",
+        "psr",
+        "psr-7",
+        "psr-11"
+    ],
+    "support": {
+        "docs": "https://docs.zendframework.com/<project-name>/",
+        "issues": "https://github.com/zendframework/<project-name>/issues",
+        "source": "https://github.com/zendframework/<project-name>",
+        "slack": "https://zendframework-slack.herokuapp.com",
+        "forum": "https://discourse.zendframework.com"
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.0-dev",
+            "dev-develop": "2.0-dev"
+        }
+    },
+    "require": {
+        "php": "^5.6 || ^7.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^6.0.8 || ^5.7.15",
```

Comment:
I agree, however PHP Unit updates twice a month or so. It should not be the goal of this template to keep track of PHP Unit releases.